README.md

```diff
@@ -22,7 +22,19 @@ jump right in to Electron development.
 npm install -g electron-forge
 electron-forge init my-new-app
 cd my-new-app
-electron-forge start
+npm start
+```
+
+Alternatively, if you have a recent enough NPM or Yarn:
```

Comment:
👍 Just so I'm on the same page as everyone else, I'm going to edit this line but keep the example invocations of `npx` and `yarn create` below, because the `yarn create` call isn't obvious.